backend/service/status/status_http.go

```diff
@@ -37,14 +37,19 @@ func (s *Service) serverStatus() rest.Handle {
 
 func (s *Service) callerStatus() rest.Handle {
 	return func(w http.ResponseWriter, r *http.Request, _ rest.Params) {
+		callerCtx := identity.ForRequest(r)
+		role := callerCtx.Identity().Role()
```

Comment:
No nil comparison here? E.g. see in status_grpc.go above.

```
role := "guest"
if callerCtx != nil {
  role = callerCtx.Identity().Role()
}
```

Reply:
in HTTP the context is always set and not nil.
In gRPC it's under development still.